lib/application/constants/images.dart

```diff
@@ -18,6 +18,9 @@ enum ImageKey {
 
   // illustrations
   logo,
+
+  // placeholders
+  user,
```

Comment:
You may remove this `// placeholders`, we just split the images between icons and logos, and the placeholder is an illustration